app/models/state_file_dependent.rb

```diff
@@ -58,7 +72,19 @@ def self.senior_cutoff_date
     MultiTenantService.statefile.end_of_current_tax_year.years_ago(65)
   end
 
+  def under_17?
+    age < 17
+  end
+
+  def over_65?
```

Comment:
Wondering if all current uses of `senior_cutoff_date` can be replaced by `over_65?`? They are effectively doing the same thing. (non-blocking, but if it's a quick effort I would appreciate it)

Reply:
I'll check!

Reply:
I was able to replace one usage. All the other usages of `senior_cutoff_date` are in tests and can't be easily replaced by `over_65?`. It's being used like a constant for setting dob on dependents in many tests so I'm hesitant to delete it. We could move it to another spot, but I'm not sure the best place to move it to.